js/actions/appActions.js

```diff
@@ -1292,6 +1292,19 @@ const appActions = {
     AppDispatcher.dispatch({
       actionType: appConstants.APP_DRAG_CANCELLED
     })
+  },
+
+  /**
+   * Notifies autoplay has been blocked
+   * @param {string} location - Location of current frame
+   * @param {number} tabId - Tab id of current frame
+   */
+  autoplayBlocked: function (location, tabId) {
+    AppDispatcher.dispatch({
+      actionType: appConstants.APP_AUTOPLAY_BLOCKED,
```

Comment:
It is for displaying origin for message box
https://github.com/brave/browser-laptop/pull/8751/files#diff-d4f09c262fdb2f86d8f7407afb926c8eR19